index.bs

```diff
@@ -1514,7 +1514,7 @@ RemoteValue = {
   WindowProxyRemoteValue //
 }
 
-InternalId = Number;
+InternalId = integer;
```

Comment:
I think you want `uint` (`integer` also allows BigInt values).

Reply:
Exactly! Thanks )